src/image_tools/bake.py

```diff
@@ -27,36 +27,22 @@ def build_image_args(version, release_version):
     """
     result = {}
 
-    if isinstance(version, dict):
-        for k, v in version.items():
-            result[k.upper()] = v
-        result["RELEASE"] = release_version
-    elif isinstance(version, str) and isinstance(release_version, str):
-        {
-            "PRODUCT": version,
-            "RELEASE": release_version,
-        }
-    else:
-        raise ValueError(f"Unsupported version object: {version}")
+    for k, v in version.items():
```

Comment:
This is breaking, right?

Reply:
This is a relict of times when versions could also be strings in addition to dicts. We don't do that anymore.

Reply:
That's fair, but I'd still give it a callout in the changelog. Especially since docker-images *still* doesn't pin image-tools.